repo2docker/app.py

```diff
@@ -326,6 +327,12 @@ def _user_name_default(self):
         config=True,
     )
 
+    s3_logs_endpoint = Unicode("", help="S3 endpoint", config=True)
+    s3_logs_access_key = Unicode("", help="S3 access key ", config=True)
+    s3_logs_secret_key = Unicode("", help="S3 secret key", config=True)
+    s3_logs_bucket = Unicode("", help="S3 bucket", config=True)
+    s3_logs_region = Unicode("", help="S3 region", config=True)
```

Comment:
I've made it into a Configurable. I decided to keep the individual options because it avoids having `aws_` in the argument names when this should work for any S3 compatible store, and will also make it easier if we want to switch libraries.